WcaOnRails/config/locales/en.yml

```diff
@@ -269,10 +269,10 @@ en:
         refund_policy_percent: "Percent to be refunded"
         refund_policy_limit_date: "Limit date for refunds"
         guests_entry_fee_lowest_denomination: "Entry fee for guests"
-        regulation_z1: "I would like to require competitors to submit puzzles during registration as per Regulation Z1)"
-        regulation_z1_reason: "Reason for Z1)"
-        regulation_z3: "I would like to select competitors who directly qualify for certain rounds of certain events based on the results of specific previous competitions as per Regulation Z3)"
-        regulation_z3_reason: "Reason for Z3)"
+        regulation_z1: "I would like to require competitors to submit puzzles at a specific time before an event is scheduled to begin"
+        regulation_z1_reason: "Reason for submitting puzzles early"
+        regulation_z3: "I would like to select competitors who directly qualify for certain rounds of certain events based on the results of specific previous competitions"
```

Comment:
Did we move Regulation Z3) to the [WCA Competition Requirements Policy](https://www.worldcubeassociation.org/documents/policies/external/Competition%20Requirements.pdf) or did we get rid of it? I did a quick check and couldn't find it.

Reply:
WCRP 5.1 about qualification times. My understanding is that qualification times are a subset of Z3 and we got rid of the rest of it (ie qualifying for non first-rounds)

Reply:
That's my understanding too, so the `regulation_z3` text should definitely be modified.

Reply:
Updated!